code_folder/A1_csv_to_json.py

```diff
@@ -0,0 +1,356 @@
+                input_directory, filename="project", parameters=parameters
+            )
+            input_json.update(single_dict)
+        elif f == "fixcost.csv":
+            parameters = [
+                "age_installed",
+                "capex_fix",
+                "capex_var",
+                "label",
+                "lifetime",
+                "opex_fix",
+                "opex_var",
+            ]
+            single_dict = create_json_from_csv(
+                input_directory, filename="fixcost", parameters=parameters
+            )
+            input_json.update(single_dict)
+        elif f == "simulation_settings.csv":
+            parameters = [
+                "display_output",
+                "evaluated_period",
+                "input_file_name",
+                "label",
+                "oemof_file_name",
+                "output_lp_file",
+                "overwrite",
+                "path_input_file",
+                "path_input_folder",
+                "path_output_folder",
+                "path_output_folder_inputs",
+                "restore_from_oemof_file",
+                "start_date",
+                "store_oemof_results",
+                "timestep",
+            ]
+            single_dict = create_json_from_csv(
+                input_directory, filename="simulation_settings", parameters=parameters
+            )
+            input_json.update(single_dict)
+        elif f == "project_data.csv":
+            parameters = [
+                "country",
+                "label",
+                "latitude",
+                "longitude",
+                "project_id",
+                "project_name",
+                "scenario_id",
+                "scenario_name",
+                "sectors",
+            ]
+            single_dict = create_json_from_csv(
+                input_directory, filename="project_data", parameters=parameters
+            )
+            input_json.update(single_dict)
+        elif f == "economic_data.csv":
+            parameters = [
+                "currency",
+                "discount_factor",
+                "label",
+                "project_duration",
+                "tax",
+            ]
+            single_dict = create_json_from_csv(
+                input_directory, filename="economic_data", parameters=parameters
+            )
+            input_json.update(single_dict)
+        elif "storage_" in f:
+            pass
+        else:
+            logging.error(
+                "The file %s" % f + " is not recognized as input file for mvs "
+                "check /mvs_eland/inputs/elements/default_csv for correct "
+                "file names."
+            )
+
+    with open(os.path.join(input_directory, output_filename), "w") as outfile:
+        json.dump(input_json, outfile, skipkeys=True, sort_keys=True, indent=4)
+    logging.info("json stored into /mvs_eland/inputs/%s", output_filename)
+    if pass_back:
+        return input_json
+
```

Comment:
ich hab ne Liste hinzugefügt, die die Namen der csv dateien als Liste einliest. Storages werden einzeln eingelesen. Allerdings z.B. nicht verschiedene eneryProduction-Typen. reicht das so?

Ein output sieht dann so aus: 
INFO:root: Json file created successully from csv's and stored into/mvs_eland/inputs/working_example2.json
Scenario includes 10 assets of type: ['storage_01', 'project', 'project_data', 'simulation_settings', 'energyProduction', 'fixcost', 'energyProviders', 'energyConsumption', 'economic_data', 'energyConversion']

Reply:
This would only confirm which csv files are read, but not which assets are created. 
For project/project_data/simulation_settings, counting would also not make sense.

Apart from counting, even better might to store the names of the created assets:

```
df = read_csv_file()
asset_name_string = " " 
for column in df:
    if column != "unit" and column != "parameters":
        asset_name_string = asset_name_string + df[column, label]
logging.info('From input file xy, following assets are added to the energy system: %s', asset_name_string)
```
I will add the needed code below. Please test if the logging message appears and if so, resolve the comment.

Reply:
what exactly do you mean with "below"? do you change it yourself?

Reply:
resolved this